fe/app/api/models/AboutUs.ts

```diff
@@ -0,0 +1,19 @@
+import { ObjectPosition } from "@components/ImageElement/ImageElement.models";
+
+export interface TeamElement {
+  name: string;
+  role: { pt: string; en: string };
```

Comment:
To avoid duplication, perhaps could you create a model for `{ pt: string; en: string }`, and also to make a standard could you also create a model for 
```
{
    teamElement_image: {
      src: string;
      objectPosition: ObjectPosition;
      title: string;
    };
  }
```

Reply:
i´ve been using **Record<Locales, string>** to this **{ pt: string; en: string }**

Reply:
yes good point!